includes/reader-activation/class-reader-data.php

```diff
@@ -264,8 +275,34 @@ public static function api_delete_item( $request ) {
 	 * @param int   $timestamp Timestamp.
 	 * @param array $data      Data.
 	 */
-	public static function set_is_newsletter_subscriber( $timestamp, $data ) {
```

Comment:
After this method is removed, [this line](https://github.com/Automattic/newspack-popups/blob/ada4246e7677ecb03213b40a32a4da8942f6a9be/includes/class-newspack-popups-segmentation.php#L75) should be also removed (or updated to check for different method).